### PR TITLE
Added form-control class for simple_form input fields

### PR DESCRIPTION
**Pull request by dbernar1**
Which, as you probably know, is the class that Bootstrap seems to want to use for most inputs.

### CODE REVIEW

File: config/initializers/simple_form_bootstrap.rb [hunk 1]
```diff
@@ -43,5 +43,4 @@
   config.form_class = "form-horizontal"
   config.label_class = "col-xs-2 col-sm-2 col-md-2 col-lg-2 control-label"
   #config.input_class = "form-control" #not yet supported
-  config.default_wrapper = :bootstrap
```
Comment (swalberg) [inline config/initializers/simple_form_bootstrap.rb:46]: What's this do, @dbernar1?

Comment (dbernar1) [inline config/initializers/simple_form_bootstrap.rb:46]: It was just doubled up. See https://github.com/winnipegrb/winnivote/blob/master/config/initializers/simple_form_bootstrap.rb

**Comment by swalberg** [comment]
aha! :ship: :it:

**Comment by dbernar1** [comment]
Hey @amirci - I presume you have some concerns about this change since you've not merged it in yet. Do you want to tell me what you're concerned about?

Thanks.

**Comment by amirci** [comment]
To be honest I am still unsure why we need this of which issue is responding to.

**Comment by dbernar1** [comment]
Thanks.

The current implementation of simple form + bootstrap is not appropriately implementing the spec for Bootstraped <input> tags.

This change makes <input> tags created by simple form have class="form-control", which is what Bootstrap asks that <input> tags must have. See http://getbootstrap.com/css/#forms-example for an example.

**Comment by amirci** [comment]
Which issue is this PR responding to?

**Comment by dbernar1** [comment]
Hi Amir,

That would be the issue about adding Bootstrap for the forms.

## 

Dan Bernardic
My contact information is at http://gravatar.com/dbernar1

**Comment by amirci** [comment]
Yes I realized that. I was expecting further explanation. No problem :)

I think is better to close it for now, if we need it in the future we can open it again.

Let's discuss next meeting.

**Pull request closed without merging**